phpBB/phpbb/install/module_base.php

```diff
@@ -110,10 +110,38 @@ public function run()
 
 		if ($task_index < $iterator->count())
 		{
-			$iterator->seek($task_index);
+		    // Receiving task index
```

Comment:
mostly because of this: https://github.com/phpbb/phpbb/blob/master/phpBB/phpbb/di/service_collection_iterator.php#L44

if it only does this with the data collector services then i think it should be fine. But who knows.